### DISCUSSION TITLE
Catalog API - Get Albums by Barcode Id No Results

### DISCUSSION THREAD

**Discussion by Zan-Aslettco** [tidal-music/discussions#56] [Q&A]
I'm trying to connect my discogs collection with tidal so that I can cross reference my physical record collection with a digital representation on tidal. It seems like the [Catalog API - Get Albums by Barcode Id](https://developer.tidal.com/apiref?spec=catalogue&ref=get-albums-by-barcode-id) should work great for this use case, but none of the barcode ids that I'm getting from discogs seem to work in this endpoint. I always get an empty response:

```
{
  "data": [],
  "metadata": {
    "requested": 0,
    "success": 0,
    "failure": 0
  }
}
```

Does anyone have any insight into where the barcode ids come from, how they should be formatted, etc..?

The data in discogs is community added, so there is a possiblity that it's incorrectly formatted or even wrong altogether. That being said, I have tried several barcode ids from discogs and none have worked for me.

**Comment by stefanmoro** [comment]
Hi @Zan-Aslettco ,

As outlined by the [API ref](https://developer.tidal.com/api-reference?spec=catalogue&ref=get-albums-by-barcode-id), barcode ids must be given in EAN-13 or UPC-A format.

I am unfortunately not familiar with discogs, or what format they use.

Thanks

**Reply by Zan-Aslettco** [reply]
Thanks for the reply!
I guess the root of my question really is what is the barcode id supposed to be tied to in order for tidal to return results. For example, a given album on discogs can have multiple different releases (vinyl, CD, USA release, UK release, re-releases, etc..). Each of these releases can have their own different barcode id. It's reasonable to expect that tidal wouldn't have record of every single release and its barcode, but do you have any additional insight into how I can find the specific release on discogs whose barcode would likely be in the tidal system?
